components/org.wso2.bfsi.consent.management.common/src/main/java/org/wso2/bfsi/consent/management/common/persistence/JDBCPersistenceManager.java

```diff
@@ -0,0 +1,155 @@
+
+    /**
+     * Initialize the data source.
+     */
+    private void initDataSource() {
+
+        if (dataSource != null) {
+            return;
+        }
+        synchronized (JDBCPersistenceManager.class) {
+            try {
+                String dataSourceName = ConsentManagementConfigParser.getInstance().getDataSourceName();
+                if (StringUtils.isNotBlank(dataSourceName)) {
+                    Context context = new InitialContext();
+                    dataSource = (DataSource) context.lookup(dataSourceName);
+                } else {
+                    throw new ConsentManagementRuntimeException("Persistence Manager configuration for Consent " +
+                            "Management is not available in bfsi-consent-management.xml file. " +
+                            "Terminating the JDBC persistence manager initialization.");
+                }
+            } catch (NamingException e) {
+                throw new ConsentManagementRuntimeException("Error when looking up the Consent Management Data Source.",
+                        e);
+            }
+        }
+    }
+
+    /**
+     * Returns an database connection for Consent Management data source.
+     *
+     * @return Database connection.
+     * @throws ConsentManagementRuntimeException Exception occurred when getting the data source.
+     */
+    public Connection getDBConnection() throws ConsentManagementRuntimeException {
+
+        try {
+            Connection dbConnection = dataSource.getConnection();
+            dbConnection.setAutoCommit(false);
+            log.debug("Returning database connection for Consent Management data source");
+            return dbConnection;
+        } catch (SQLException e) {
+            throw new ConsentManagementRuntimeException("Error when getting a database connection object from the " +
+                    "consent management data source.", e);
+        }
+    }
+
+    /**
+     * Returns Consent Management data source.
+     *
+     * @return Data source.
+     */
+    public DataSource getDataSource() {
+
+        return dataSource;
+    }
+
+    /**
+     * Revoke the transaction when catch then sql transaction errors.
+     *
+     * @param dbConnection database connection.
+     */
+    public void rollbackTransaction(Connection dbConnection) {
+
+        try {
+            if (dbConnection != null) {
+                dbConnection.rollback();
+            }
+        } catch (SQLException e) {
+            log.error("An error occurred while rolling back transactions. ", e);
+        }
+    }
+
+    /**
+     * Commit the transaction.
+     *
+     * @param dbConnection database connection.
+     */
+    public void commitTransaction(Connection dbConnection) {
+
+        try {
+            if (dbConnection != null) {
+                dbConnection.commit();
+            }
+        } catch (SQLException e) {
+            log.error("An error occurred while commit transactions. ", e);
+        }
+    }
```

Comment:
DatabaseUtil class contains same set of methods

Reply:
Removed the databaseutil class